src/components/dashboard/BackToDealLink.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Button } from '../ui/button'
+
+export interface BackToDealLinkProps {
+  id: string
+}
+
+export default function BackToDealLink({ id }: BackToDealLinkProps) {
+  return (
+    <Button
```

Comment:
I think for this to be accessible you need to add the `asChild` attribute to the button.
```suggestion
    <Button asChild
```

Reply:
Do you know what the implication is specifically for that?

Reply:
> Do you know what the implication is specifically for that?

You can disregard this, as per your other comment on this. Thanks, James!